persistence/shared_sql.go

```diff
@@ -4,13 +4,15 @@ import (
 	"context"
 	"encoding/hex"
 	"fmt"
+
 	"github.com/jackc/pgx/v4"
 
 	"github.com/pokt-network/pocket/persistence/schema"
 	"github.com/pokt-network/pocket/shared/types"
 )
 
-// IMPROVE(team): Move this into a proto enum
+// IMPROVE(team): Move this into a proto enum. We are not using `iota` for the time being
```

Comment:
(Non-Actionable) StakeStatus is deprecated at the state level in #154 expect more cleanup to come here

Reply:
ACK